aspects/aspects.go

```diff
@@ -444,9 +444,17 @@ type expandedMatch struct {
 	value interface{}
 }
 
+// maxValueDepth is the limit on a value's nestedness. Since creating a highly
+// nested JSON value takes only few bytes per level but makes snapd recurse
+// creating a 2Kb stack, we need to prevent this from being abused.
+var maxValueDepth = 4096
```

Comment:
Should we not perhaps be even more restrictive to start off with? It easier to become less restrictive as required than other way round.

Reply:
Sure, that's a fair point, 256/512 then?

Reply:
+1

Reply:
I would say 256

Reply:
Sounds good, changed it 64 now. Thank you